psydac/linalg/utilities.py

```diff
@@ -84,6 +84,11 @@ def petsc_to_psydac(x, Xh):
     x : PETSc.Vec
       PETSc vector
 
+    Xh: psydac.linalg.stencil.StencilVectorSpace | psydac.linalg.block.BlockVectorSpace
+      Space of the coefficients of the Psydac vector.
+
+    out: Psydac vector to store the result.
```

Comment:
More complete and precise, and consistent with the rest of the docstring:
```suggestion
    Xh : psydac.linalg.stencil.StencilVectorSpace | psydac.linalg.block.BlockVectorSpace
      Space of the coefficients of the Psydac vector.

    out : psydac.linalg.stencil.StencilVector | psydac.linalg.block.BlockVector, optional
      The Psydac vector where to store the result.
``````